Description: update the upstream homepage in program help and manpage.
Author: Joao Eriberto Mota Filho <eriberto@debian.org>
Last-Update: 2015-12-14
Index: packit-1.0/man/packit.8.in
===================================================================
--- packit-1.0.orig/man/packit.8.in
+++ packit-1.0/man/packit.8.in
@@ -946,6 +946,6 @@ The latest version can be found at:
 .br
 .ti +7
 .ft I
-http://packit.sourceforge.net
+http://packetfactory.openwall.net/projects/packit
 .ft
 
Index: packit-1.0/src/version.h.in
===================================================================
--- packit-1.0.orig/src/version.h.in
+++ packit-1.0/src/version.h.in
@@ -22,5 +22,5 @@
 
 #define P_VERSION "@PACKIT_VERSION@"
 #define P_AUTHOR "Darren Bounds <dbounds@intrusense.com>"
-#define P_SITE "http://packit.sourceforge.net"
+#define P_SITE "http://packetfactory.openwall.net/projects/packit"
 
